src/Fixer/Whitespace/ArrayIndentationFixer.php

```diff
@@ -35,6 +39,7 @@ public function getDefinition(): FixerDefinitionInterface
             'Each element of an array must be indented exactly once.',
             [
                 new CodeSample("<?php\n\$foo = [\n   'bar' => [\n    'baz' => true,\n  ],\n];\n"),
```

Comment:
TBH I don't get what this option does, are you able to update the samples to have the same input code with both `strict` options to see the difference?

Reply:
I updated the code sample, it more clear now ?

The difference is the same than
`SINGLE_SPACE` vs `AT_LEAST_SINGLE_SPACE` for the `BinaryOperatorSpacesFixer`.

Currently, `array_indentation` enforce EXACTLY one indentation.
When looking at PSR12, it enforce indentation but not "exactly" one, so it could be considered to add the rule with the `strict => false` option to enforce AT LEAST one indentation.

This option would be also useful for SymfonyRuleset since they use indentation, but sometimes use more than one indentation.

In my case this would be useful for https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7813 because I need a fixer to resolve indentation after mine but the `array_indentation` is too strict atm to be added in the PSR ruleset.